README.md

```diff
@@ -183,6 +187,34 @@ for epoch in range(50):
 
 Currently, PyTorch Frame support the following semantic types: `numerical`, `categorical`, `multicategorical`, `text_embedded`. [Here](https://github.com/pyg-team/pytorch-frame/blob/master/examples/mercari.py) is an example of how to train a model on dataset with all four semantic types.
 
+### Integration with Large Language Models
+
+PyTorch Frame supports integration with existing embeddings generated by Large Language Model. With any downloaded model or embedding API endpoints, you can encode your text data with embeddings and train it with deep learning models alongside other semantic types.
+
+We support the following (but not limited to):
+<table>
+  <tr>
+    <td align="center">
+      <a href="https://platform.openai.com/docs/guides/embeddings">
+        <img src="docs/source/_figures/OpenAI_Logo.png" alt="OpenAI" width="100px"/>
+      </a>
+      <br /><a href="https://github.com/pyg-team/pytorch-frame/blob/master/examples/openai_embedding.py">OpenAI Embedding Code Example</a>
+    </td>
+    <td align="center">
+      <a href="https://cohere.com/embeddings">
+        <img src="docs/source/_figures/cohere-logo.png" alt="Cohere" width="100px"/>
+      </a>
+      <br /><a href="https://github.com/pyg-team/pytorch-frame/blob/master/examples/openai_embedding.py">Cohere Embed v3 Code Example</a>
```

Comment:
Rename the `openai_embedding.py` to `text_api_embeddings.py` as mentioned by Weihua and change both openai and cohere example path to that?

Reply:
I want to call it `llm_embeddings.py` if that's fine with you

Reply:
I am ok. Though it sounds too fancy to me 😂